client/src/components/Jobs/Jobs.jsx

```diff
@@ -1,23 +1,38 @@
-import React, { useCallback } from 'react';
+import React, { useCallback, useEffect } from 'react';
 import PropTypes from 'prop-types';
-import { useDispatch, useSelector } from 'react-redux';
+import { useDispatch, shallowEqual, useSelector } from 'react-redux';
 import { Link } from 'react-router-dom';
 import { AppIcon, InfiniteScrollTable, Message } from '_common';
 import { formatDateTime } from 'utils/timeFormat';
 import { getOutputPath } from 'utils/jobsUtil';
 import JobsStatus from './JobsStatus';
 import './Jobs.scss';
 import * as ROUTES from '../../constants/routes';
+import Searchbar from '_common/Searchbar';
+import queryStringParser from 'query-string';
+import { useLocation } from 'react-router-dom';
 
-function JobsView({ showDetails, showFancyStatus, rowProps }) {
+function JobsView({
+  showDetails,
```

Comment:
I noticed that clicking on "View Details" on a job when a search is active will reset the search. For example:
1. First, I search and get results:
<img width="1275" alt="Screen Shot 2023-02-21 at 11 07 33 AM" src="https://user-images.githubusercontent.com/20326896/220413763-b320f0f2-22a4-488a-bb91-6d2092291fba.png">
2. Then, I click on "View Details", and the listing behind the modal resets, clearing the search results:
<img width="1325" alt="Screen Shot 2023-02-21 at 11 07 42 AM" src="https://user-images.githubusercontent.com/20326896/220413761-fa5daf69-cb70-48f5-a21e-051ed06d8937.png">